salt/_modules/metalk8s.py

```diff
@@ -11,10 +11,6 @@
 
 KUBERNETES_PRESENT = False
 try:
-    import kubernetes.client
```

Comment:
So no more K8s imports? Then the whole `KUBERNETES_PRESENT` and `@depends` on it etc can be removed, no?

Reply:
we can still use the `@depends` as the `wait_apiserver` function still need the `metalk8s_kubernetes` salt module not the kubernetes import, but true, my `try...except` bloc here does not make sense, will fix it